jmix-superset/superset-flowui/src/main/resources/ui.xsd

```diff
@@ -0,0 +1,52 @@
+<!--
+  ~ Copyright 2024 Haulmont.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<xs:schema targetNamespace="http://jmix.io/schema/superset/ui"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:layout="http://jmix.io/schema/flowui/layout"
+           xmlns="http://jmix.io/schema/superset/ui"
+           elementFormDefault="qualified">
+    <xs:element name="dashboard">
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="layout:baseComponent">
+                    <xs:sequence minOccurs="0">
+                        <xs:element name="datasetConstraints" type="datasetConstraintsType"/>
+                    </xs:sequence>
+
+                    <xs:attribute name="embeddedId" type="xs:string"/>
+                    <xs:attribute name="url" type="xs:string"/>
```

Comment:
What is the point in having URL property in XML and in public Java setter if:
* we cannot define username and password for this URL
* access token in TokenManager is anyway taken from the Superset instance configured using application properties?

Reply:
I thought about a case when application will have its own mechanism of getting/refreshing tokens that is not rely on add-on's implementation. In this case, you can set URL for component because all tokens are handled by application.

Reply:
It was decided to remove URL property from component. Fixed.